oddt/shape.py

```diff
@@ -0,0 +1,122 @@
+import numpy as np
+import scipy.stats as stats
+
+
+def calculate_standard_usr(atoms):
```

Comment:
The main argument should be `molecule`, an instance of `oddt.toolkit.Molecule` class. You can capture atoms from atom_dict or whatever to atom variable below.